.github/workflows/kind-e2e.yaml

```diff
@@ -66,7 +66,8 @@ jobs:
 
     env:
       GOPATH: ${{ github.workspace }}
-      KO_DOCKER_REPO: kind.local
+      KO_DOCKER_REPO: localhost:5001
+      REGISTRY_PORT: 5001
```

Comment:
Why this change was necessary?

Reply:
Because `kind.local` is `ko` only feature and it does some magic with `KinD` (it basically does the equivalent of `kind load image ...`), so we need a proper local container registry